### DISCUSSION TITLE
[24 Gennaio 2023] Modifica CPU

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/architettura-degli-elaboratori#68] [Modifiche architettura MIPS (M-Z)]
![Esame](https://github.com/Elia-Belli/ae2223/assets/107944476/621a87b4-be6e-4ca3-9731-0dbac33e3074)
![Esame](https://github.com/Elia-Belli/ae2223/assets/107944476/415a7a37-4221-4d90-ae39-0bbe1a842917)
![CPU](https://github.com/Elia-Belli/ae2223/assets/107944476/baa90459-d6ce-4f02-8d74-e650f81c96d9)
---
### Soluzione del Prof
Le soluzioni fornite dal professore potrebbero essere incomplete o poco chiare, quindi sono benvenute soluzioni che le completano, oppure soluzioni alternative e/o con spiegazioni più approfondite: 
- [Disegno.pdf](https://github.com/sapienzastudentsnetwork/archelab2223/files/11666008/AE.2021-22.-.Appello.4.-.2023-01-24.-.Esercizio.CPU.-.Diagramma.pdf)
- [Soluzione.pdf](https://github.com/sapienzastudentsnetwork/archelab2223/files/11666017/AE.2021-22.-.Appello.4.-.2023-01-24.-.Esercizio.CPU.-.Scheda.di.calcolo.pdf)

**Comment by Frisbon** [comment]
![image](https://github.com/sapienzastudentsnetwork/archelab2223/assets/84731341/a3e1651e-dfad-46c7-be2c-d2ca31dba393)
![image](https://github.com/sapienzastudentsnetwork/archelab2223/assets/84731341/cf3cc3ff-70c9-4500-8b5c-c6180170e406)
![image](https://github.com/sapienzastudentsnetwork/archelab2223/assets/84731341/b439d357-64d5-4f0f-b68a-93edf8280e9e)


## Spero sia giusto... ecco
